src/Core/Services/Implementations/UserService.cs

```diff
@@ -392,6 +392,20 @@ public async Task SendMasterPasswordHintAsync(string email)
         await _mailService.SendMasterPasswordHintEmailAsync(email, user.MasterPasswordHint);
     }
 
+
+    public async Task<bool> SendRequestAccessToSM(Guid organizationId, User user, string emailContent)
```

Comment:
Based on @jlf0dev 's feedback, I would recommend moving this out into its own command that the SM team would own. 

Probably somewhere in `bitwarden_license/src/Commercial.Core/SecretsManager/Commands`